pkgs/racket-doc/scribblings/reference/vectors.scrbl

```diff
@@ -485,22 +485,27 @@ Like @racket[vector-member], but finds an element using @racket[eq?].
  not modified). This sort is stable (i.e., the order of ``equal''
  elements is preserved).
 
+ If @racket[end] is @racket[#f], it is replaced with
+ @racket[(vector-length vec)].
+
 @mz-examples[#:eval vec-eval
 (define v1 (vector 4 3 2 1))
-(vector-sort v1 <)
 v1
```

Comment:
I think the point of having `v1` after `(vector-sort v1 <)` originally is to show whether `vector-sort` has side-effect. So we should either move `v1` to the end of the sequences of `(vector-sort v1 ....)`. Alternatively, if you don't think it's worth the effort to do the demonstratation, then we should just delete the lone `v1` / `v2`.